src/components/Panel/Panel.js

```diff
@@ -29,6 +46,21 @@ class Panel extends Component {
           />
         )}
 
+        {collapsible && (
+          <button
+            className={cssClasses.collapseButton}
+            aria-expanded={!this.state.collapsed}
+            onClick={() => {
```

Comment:
We don't `preventDefault()` which means that the button might trigger an action e.g. nested inside a form.